components/List/List.Styled.js

```diff
@@ -0,0 +1,57 @@
+import styled, { css } from "styled-components";
+
+export const TransactionsList = styled.ul`
+  padding: 0;
+  margin: 0 0.5rem 0 0.5rem;
+  list-style: none;
+`;
+export const TransactionCard = styled.li`
+  position: relative;
+  justify-content: space-between;
+  gap: 0.8rem;
+  border-radius: 5px;
+  margin: 10px 0px;
+  padding: 7px;
+  box-shadow: rgba(0, 0, 0, 0.2) 0px 2px 1px -1px,
+    rgba(0, 0, 0, 0.14) 0px 1px 1px 0px, rgba(0, 0, 0, 0.12) 0px 1px 3px 0px;
+`;
```

Comment:
The Flexbox properties `justify-content` and `gap` will have no effect because the element isn't defined as a flexbox container.

```suggestion
export const TransactionCard = styled.li`
  position: relative;
  border-radius: 5px;
  margin: 10px 0px;
  padding: 7px;
  box-shadow: rgba(0, 0, 0, 0.2) 0px 2px 1px -1px,
    rgba(0, 0, 0, 0.14) 0px 1px 1px 0px, rgba(0, 0, 0, 0.12) 0px 1px 3px 0px;
`;
```